src/Agent.Worker/Handlers/NodeHandler.cs

```diff
@@ -333,6 +345,28 @@ private void OnDataReceived(object sender, ProcessDataReceivedEventArgs e)
             }
         }
 
+        private bool isNode20SupportedSystems() {
```

Comment:
>bool [](http://example.com/codeflow?start=16&length=4)

suggest collecting telemetry on detected OS / version for validation

Reply:
Updated: added Trace.Info() for detected system os/ verison.